### PR TITLE
Seperate stdout from stderr on logs.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: logs/logWrapper.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+type stdoutWriter struct {
+	writer io.Writer
+}
+
+type stderrorWriter struct {
+	writer io.Writer
+}
+type stdbothWriter struct {
```
Comment (ibuildthecloud) [inline logs/logWrapper.go]: not used?

File: logs/logs.go [hunk 2]
```diff
@@ -31,8 +31,6 @@ func GetLogs(rw http.ResponseWriter, req *http.Request) error {
 
 	if found {
 		tail = strconv.Itoa(int(tailTemp))
```
Comment (ibuildthecloud) [inline logs/logs.go]: Is the 100 not needed because we assume the token has the default?

File: logs/logWrapper.go (hunk 1 above)
Comment (hibooboo2) [inline logs/logWrapper.go]: Was going to use it for the tty case for got to ask you if you would prefer that?
That way you can tell you dont have differentiated logs.

File: logs/logs.go (hunk 2 above)
Comment (hibooboo2) [inline logs/logs.go]: Defensive programming.

Comment (ibuildthecloud) [inline logs/logs.go]: No, this is now non-defensive... Not saying you should change, was just curious on why?

File: logs/logWrapper.go (hunk 1 above)
Comment (ibuildthecloud) [inline logs/logWrapper.go]: That's a good point.  Lets try that and see if vince complains.

File: logs/logs.go (hunk 2 above)
Comment (hibooboo2) [inline logs/logs.go]: Well i thought the 100 was there lol. Didnt realize i deleted it. I had it in 2 places. You are right i derped. Been a rough day

File: logs/logWrapper.go (hunk 1 above)
Comment (hibooboo2) [inline logs/logWrapper.go]: k

**Pull request merged**